docs/data-sources/service_account.md

```diff
@@ -13,21 +13,25 @@ Data Source representing a Prefect Service Account
 ## Example Usage
 
 ```terraform
-data "prefect_service_account" "my_existing_bot" {
-  id = "service-acount-UUID"
+data "prefect_service_account" "bot" {
+  id = "7de0291d-59d0-4d57-a629-fe47960aa61b"
```

Comment:
Do we want this?

Reply:
yeah it should be fine, this is for the docs pages and that's a random uuid i created just now